### PR TITLE
Update command to retrieve admin key in base 64

### PR DESCRIPTION
**Pull request by sgrasley**
Original command was not creating correct base64 key.

### CODE REVIEW

File: doc/source_tech_preview/deployment/configure.rst [hunk 1]
```diff
@@ -118,7 +118,7 @@ The Ceph admin keyring can be obtained by running the following on ceph host.
 
 .. code-block:: yaml
 
-  sudo ceph auth get-key client.admin | base64
+  echo $( sudo ceph auth get-key client.admin ) | base64
```
Comment (jesusaurus) [inline doc/source_tech_preview/deployment/configure.rst]: I think this could be further improved by using `base64 <(sudo ceph auth get-key client.admin)`

Comment (jgu17) [reply doc/source_tech_preview/deployment/configure.rst]: could you explain advantage over "echo"?

Comment (sgrasley) [reply doc/source_tech_preview/deployment/configure.rst]: adds a \n to string that is getting base64 encoded.  had issues authenticating with ses without /n.

Comment (jgu17) [reply doc/source_tech_preview/deployment/configure.rst]: sorry the question was for @jesusaurus.

Comment (jesusaurus) [reply doc/source_tech_preview/deployment/configure.rst]: It's not a very important difference, but it spawns fewer processes/subshells by giving the base64 process a direct file handle for the ceph output.

**Comment by csymons-suse** [comment]
superseded by https://github.com/SUSE-Cloud/socok8s/pull/383
change transferred to doc/source

**Pull request closed without merging**